source/frontend/src/features/mapSideBar/acquisition/common/GenerateForm/hooks/useGenerateAgreement.test.tsx

```diff
@@ -111,7 +134,30 @@ describe('useGenerateAgreement functions', () => {
     await act(async () => {
       await generate(mockAgreementsResponse()[1]);
       expect(generateFn).toHaveBeenCalled();
-      expect(getOrganizationConceptFn).toHaveBeenCalled();
+      expect(getOrganizationConceptFn).toHaveBeenCalledWith(1);
+      expect(getAcquisitionFileProperties).toHaveBeenCalled();
+    });
+  });
+
+  it('makes requests to expected api endpoints if a team member is a negotiating agent with person', async () => {
+    const responseWithTeam: Api_AcquisitionFile = {
+      ...mockAcquisitionFileResponse(),
+      acquisitionTeam: [
+        {
+          id: 1,
+          acquisitionFileId: 1,
+          organizationId: 1,
+          teamProfileTypeCode: 'NEGOTAGENT',
+          rowVersion: 2,
+        },
+      ],
+    };
+    const generate = setup({ acquisitionResponse: responseWithTeam });
+
+    await act(async () => {
+      await generate(mockAgreementsResponse()[1]);
+      expect(generateFn).toHaveBeenCalled();
+      expect(getPersonConceptFn).toHaveBeenCalledWith(1);
       expect(getAcquisitionFileProperties).toHaveBeenCalled();
```

Comment:
not sure I follow this test. We already have a test for negotiating agent being a person so I would expect this test for an org - right?
If it is for an org, then the test description should be changed to "... negotiating agent with org" and also the test assertions should change to `expect(getOrganizationConceptFn).toHaveBeenCalledWith(1);` instead of `expect(getPersonConceptFn).toHaveBeenCalledWith(1);`